musl support

```shell
In file included from /sherpa-onnx/build/_deps/kaldifst-src/kaldifst/csrc/text-utils.cc:8:
/sherpa-onnx/build/_deps/kaldifst-src/kaldifst/csrc/text-utils.h: In function 'bool kaldifst::ConvertStringToInteger(const std::string&, Int*)':
/sherpa-onnx/build/_deps/kaldifst-src/kaldifst/csrc/text-utils.h:48:3: error: 'int64_t' was not declared in this scope
   48 |   int64_t i = KALDIFST_STRTOLL(this_str, &end);
      |   ^~~~~~~
/sherpa-onnx/build/_deps/kaldifst-src/kaldifst/csrc/text-utils.h:14:1: note: 'int64_t' is defined in header '<cstdint>'; this is probably fixable by adding '#include <cstdint>'
   13 | #include <vector>
  +++ |+#include <cstdint>
   14 | 
/sherpa-onnx/build/_deps/kaldifst-src/kaldifst/csrc/text-utils.h:52:31: error: 'i' was not declared in this scope
   52 |   Int iInt = static_cast<Int>(i);
      |                               ^
/sherpa-onnx/build/_deps/kaldifst-src/kaldifst/csrc/text-utils.h:53:19: error: 'int64_t' does not name a type
   53 |   if (static_cast<int64_t>(iInt) != i ||
      |                   ^~~~~~~
/sherpa-onnx/build/_deps/kaldifst-src/kaldifst/csrc/text-utils.h:53:19: note: 'int64_t' is defined in header '<cstdint>'; this is probably fixable by adding '#include <cstdint>'
/sherpa-onnx/build/_deps/kaldifst-src/kaldifst/csrc/text-utils.h: In function 'bool kaldifst::SplitStringToIntegers(const std::string&, const char*, bool, std::vector<I>*)':
/sherpa-onnx/build/_deps/kaldifst-src/kaldifst/csrc/text-utils.h:107:5: error: 'int64_t' was not declared in this scope
  107 |     int64_t j = 0;
      |     ^~~~~~~
/sherpa-onnx/build/_deps/kaldifst-src/kaldifst/csrc/text-utils.h:107:5: note: 'int64_t' is defined in header '<cstdint>'; this is probably fixable by adding '#include <cstdint>'
/sherpa-onnx/build/_deps/kaldifst-src/kaldifst/csrc/text-utils.h:108:5: error: 'j' was not declared in this scope
  108 |     j = KALDIFST_STRTOLL(this_str, &end);
      |     ^
/sherpa-onnx/build/_deps/kaldifst-src/kaldifst/csrc/text-utils.h:114:23: error: 'int64_t' does not name a type
  114 |       if (static_cast<int64_t>(jI) != j) {
      |                       ^~~~~~~
/sherpa-onnx/build/_deps/kaldifst-src/kaldifst/csrc/text-utils.h:114:23: note: 'int64_t' is defined in header '<cstdint>'; this is probably fixable by adding '#include <cstdint>'
gmake[2]: *** [_deps/kaldifst-build/kaldifst/csrc/CMakeFiles/kaldifst_core.dir/build.make:205: _deps/kaldifst-build/kaldifst/csrc/CMakeFiles/kaldifst_core.dir/text-utils.cc.o] Error 1
gmake[1]: *** [CMakeFiles/Makefile2:2254: _deps/kaldifst-build/kaldifst/csrc/CMakeFiles/kaldifst_core.dir/all] Error 2
gmake: *** [Makefile:156: all] Error 2
```

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -17,7 +17,7 @@ cmake_minimum_required(VERSION 3.13 FATAL_ERROR)
 
 project(kaldifst CXX)
 
-set(KALDIFST_VERSION "1.7.16")
+set(KALDIFST_VERSION "1.7.17")
 
 # Disable warning about
 #

--- a/kaldifst/csrc/text-utils.h
+++ b/kaldifst/csrc/text-utils.h
@@ -7,6 +7,7 @@
 
 #include <errno.h>
 
+#include <cstdint>
 #include <limits>
 #include <string>
 #include <type_traits>

